lib/resources/r4/patient.yaml

```diff
@@ -209,6 +209,7 @@ fields:
       <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
       <li> Only one `prefix` and one `suffix` may be provided. </li>
       <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+      <li> `family`, `given`, `prefix`, and `suffix` have a character limit of 100. </li> 
```

Comment:
we don't need to wrap the html content in spaces inside the li tags. can you fix this throughout the file?

(looks like this extra space makes it into our rendered html, even though it doesn't affect how the html looks, i would like to remove it)

Reply:
https://github.com/cerner/fhir.cerner.com/pull/414/commits/5a9a9c3bf70e3347b6761980185cdc7219853baa